orchagent/dash/dashorch.h

```diff
@@ -16,28 +18,15 @@
 #include "zmqorch.h"
 #include "zmqserver.h"
 
-struct ApplianceEntry
-{
-    swss::IpAddress sip;
-    uint32_t vm_vni;
-};
-
-struct RoutingTypeEntry
-{
-    std::string action_name;
-    std::string action_type;
-    std::string encap_type;
-    uint32_t vni;
-};
+#include "dash_api/appliance.pb.h"
+#include "dash_api/route_type.pb.h"
+#include "dash_api/eni.pb.h"
+#include "dash_api/qos.pb.h"
 
 struct EniEntry
 {
     sai_object_id_t eni_id;
-    std::string mac_address;
-    std::string qos_name;
-    swss::IpAddress underlay_ip;
-    bool admin_state;
-    std::string vnet;
+    dash::eni::Eni metadata;
 };
 
 struct QosEntry
```

Comment:
We can possibly remove this too right?

Reply:
Yes, it should be removed because it wasn't used anywhere. It's my careless mistake